src/app/public/modules/radio/radio.component.ts

```diff
@@ -158,6 +222,9 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     }
   }
 
+/**
+ * Fires when users select a radio button.
+ */
   @Output()
```

Comment:
We need a JSDoc comment to describe this event since it shows up in the docs. The existing docs include the description `Fires when users select a radio button.`

Reply:
Looks like this wasn't addressed. I think we definitely need to make this change.

Reply:
I should've added a link to the issue. The docs are added, but they won't render. Alex filed this [issue](https://github.com/blackbaud/skyux-docs-tools/issues/68) for it.